packages/ComponentLibrary/src/components/Table/tableConstants.ts

```diff
@@ -3,11 +3,22 @@ export const TOOLTIPS = {
   VIEWS: 'Views',
```

Comment:
Is it correct to use English values ​​as constants? Wouldn't they be mocks? In any case, they should be parameters so they can be translated